src/plugins/dashless/index.ts

```diff
@@ -0,0 +1,48 @@
+import { ReactNative } from "src/metro/common/libraries/index.ts";
+import { after } from "src/patcher/index.ts";
+import { definePlugin } from "#plugin-context";
+import { Dev } from "@data/constants";
+
+const { View } = ReactNative;
+
+const traverseAndModify = (node: any): any => {
+  if (typeof node === "string") {
+    return node.replace(/-/g, " ");
+  }
+  if (Array.isArray(node)) {
+    return node.map(traverseAndModify);
+  }
+  if (node && typeof node === "object" && node.props && node.props.children) {
+    return {
+      ...node,
+      props: {
+        ...node.props,
+        children: traverseAndModify(node.props.children),
+      },
+    };
+  }
+  return node;
+};
+
+let unpatchRender: () => void;

```

Comment:
Wintry provides a scoped patcher. Use it instead to ensure automatic unpatching during the plugin cleanup cycle:
```ts
import { patcher } from "#plugin-context";
```